src/blocks/social-profiles/inspector.js

```diff
@@ -73,10 +76,15 @@ class Inspector extends Component {
 		];
 
 		const defaultColors = [
+			{
+				value: blockBackgroundColor.color,
+				onChange: setBlockBackgroundColor,
+				label: __( 'Block Background Color' ),
+			},
 			{
 				value: backgroundColor.color,
 				onChange: setBackgroundColor,
-				label: __( 'Background Color' ),
+				label: __( 'Button Background Color' ),
```

Comment:
Same as in the share block. This should be "Button Color" and "Text Color" should change depending on which style is selected: ("Icon Color", "Text Color", "Icon and Text Color")

Reply:
Adjusted social profiles to use dynamic labels depending on icon style. Also removed contrast checker for block background.